### ISSUE TITLE
Sizes of Dataframes are not bounded in the new documenter

### ISSUE THREAD

**Issue by Datseris** [JuliaDocs/Documenter.jl#1205]
Following the chat on Slack with @fredrikekre , what happens is that if one produces `DataFrame` in the docs with 1000s of rows, they are not trunctuated:

![data_frame_expansion_documenter](https://user-images.githubusercontent.com/19669089/69901598-28d6d480-1384-11ea-935f-3ff65c4e2806.PNG)

This may lead to a docpage having literally millions of rows :D

**Comment by fredrikekre** [comment]
We should probably set displaysize, and possible compact, in the io. I don't think this has anything to do with the new Documenter though.

**Comment by mortenpi** [comment]
This looks like the `test/html` MIME though. Will that even be affected by `displaysize` etc?

**Comment by fredrikekre** [comment]
Thats true. We could at least set those values and then this issue can move to DataFrames.jl :trollface: 

Also, @Datseris you might be interested in https://github.com/tkf/DisplayAs.jl if some other mimetype is more appropriate.

**Comment by Datseris** [comment]
Well, I could use that, but which mime type is documenter using, and which mime type is the "console"?

Dataframes print fine on the console, but not on documenter. So I guess what I have to do is "print them as in the console". But I don't really know "how"...

**Comment by fredrikekre** [comment]
`text/plain`, but I think it might require Documenter to pass compact etc, but you can try.
